packages/nextjs/services/database/config/schema.ts

```diff
@@ -101,6 +99,124 @@ export const stagesRelations = relations(stages, ({ one, many }) => ({
   approvalVotes: many(approvalVotes),
 }));
 
+export const largeGrants = pgTable("large_grants", {
+  id: serial("id").primaryKey(),
+  grantNumber: integer("grant_number").notNull().default(1),
+  title: varchar("title").notNull(),
+  description: text("description").notNull(),
+  showcaseVideoUrl: text("showcaseVideoUrl"),
+  requestedFunds: bigint("requestedFunds", { mode: "bigint" }).notNull(),
+  github: text("github").notNull(),
+  email: text("email").notNull(),
+  twitter: text("twitter"),
+  telegram: text("telegram"),
+  submitedAt: timestamp("submited_at").default(sql`now()`),
```

Comment:
@Pabl0cks I realize there are a typo in all **submitedAt** fields (grants and stages tables). It should be **submittedAt**, with two **Ts**. 

I'm not sure to fix it on largeGrants because it will be a PITA for some shared components...

What do you think?

Reply:
Umm I think its keep it similar to small grants with spelling mistake :( just increase for future 🙌